src/components/DetailsResults/DetailsResults.js

```diff
@@ -168,7 +168,7 @@ const DetailsResults = ({ allowSortBy, defaultSortBy, defaultDirection, excludeS
                       className="data-ellipsis"
                       template={<TextTooltipTemplate text={resultValue} />}
                     >
-                      {resultValue}
+                      {resultValue.toString()}
```

Comment:
if `resultValue` is `null` or `undefined` we will get an error. 
```suggestion
                      {String(resultValue ?? '')}
```